include/sound/soc-topology.h

```diff
@@ -44,6 +44,7 @@ enum snd_soc_dobj_type {
 	SND_SOC_DOBJ_DAI_LINK,
 	SND_SOC_DOBJ_CODEC_LINK,
 	SND_SOC_DOBJ_WIDGET,
+	SND_SOC_DOBJ_ROUTE,
```

Comment:
we should really reorder/cleanup this list to make it clear which of the dobj have a match in include/uapi/sound/asoc.h  and which ones are internal to the topology code.

Reply:
@plbossart i've re-ordered this list and renamed ROUTE to GRAPH to align with the topology header type. I suppose there will be a conflict with Bard's PR but we can fix it depending on which order the PR's get merged